deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml

```diff
@@ -78,6 +94,7 @@ spec:
           required:
           - backingServiceSelector
           - applicationSelector
+          - customEnvVar
```

Comment:
Is there a reason why customEnvVar is declared as required? In cases where a volume mount is used exclusively, there would be no need to define custom environment variables, right?